index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <title>Amel Portfolio</title>
+</head>
+
+<body>
+  <header class="header-class">
+    <nav class="nav-container">
+      <a href=# class="flex-logo"><img src="./img/My-Logo.png"  alt="logo"></a>
```

Comment:
Optional suggestion

- [ ] Well done here to use `a tag` as requested, however there is a typo in the href=#, it should be `href="#"`